scripts/examples/50-Arduino-Boards/Nano-33-BLE-Sense/53-Thermal/thermal_camera.py

```diff
@@ -11,25 +11,16 @@
 import time
 import fir
 
+IMAGE_SCALE = 5.0  # Higher scaling uses more memory.
```

Comment:
```suggestion
IMAGE_SCALE = 5  # Scale image to 5x.
```

Should be the same everywhere, no need for float.

Reply:
Done